backend/tests/integration/records/views/test_views_jobs.py

```diff
@@ -149,7 +149,7 @@ def test_jobs_accelerator_experiments(inspire_app, datadir):
 
 
 def test_jobs_record_search_results_does_not_return_pending_job_to_non_superuser(
-    inspire_app
+    inspire_app,
```

Comment:
you don't need the extra comma

Reply:
That's pre-commit thing, it's added automatically